css/index.css

```diff
@@ -143,4 +135,179 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* Start of my css code */
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    flex-flow: row wrap;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    
+}
+
+.header nav a {
+    margin-right: 70px;
+    text-decoration: none;
+    color: black;
+    
+}
+
+.header nav a:last-child {
+    margin-right: 0px;
+}
+
+.header nav {
+   
+    margin-top: 20px;
+}
+
+.jumbo {
+   width: 100%;
+   margin-top: 20px;
+}
+
+.middle-content .boxes .box1 {
```

Comment:
Instead of repeating the same thing have `.middle-content .boxes` with all the repetitive styles then on `.middle-content .boxes .box1` only change the color